analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -160,9 +169,16 @@ def perform_analysis(args, skip_handlers, actions, metadata_tool,
                       "the Clang Static Analyzer.")
             return
 
-    actions = prepare_actions(actions, analyzers)
     config_map = analyzer_types.build_config_handlers(args, analyzers)
 
+    no_checker_analyzers = \
+        [a for a in analyzers if not __has_enabled_checker(config_map[a])]
+    for analyzer in no_checker_analyzers:
+        LOG.info("No checkers enabled for %s", analyzer)
```

Comment:
Can we tell if the user explicitly enabled an analyzer? If so, we should hard error, shouldn't we?

Reply:
I can be convinced, but I would feel a hard error a little strong here.

Reply:
I'll give you that one.